h09centrumspreiding.Rmd

```diff
@@ -815,15 +815,15 @@ Klik in de bovenbalk op `Descriptives`. Selecteer vervolgens de gewenste variabe
 
 Als `R` is aangeklikt als optie om de nieuwe variabele te definiëren, verschijnt er boven de data een veld met "#Enter your R code here :)". Hier kan R code worden gegeven die met behulp van R functies de nieuwe variabele definieert. Om een standaardscore te berekenen kan de volgende code worden gebruikt (hier voor de variabele 'Lezen'):
 ```
 ((Lezen - mean(Lezen)) / sd(Lezen))
 ```
 Vul de R code in en klik onderaan op `Compute column` om de lege variabele te vullen met de standaardscores.
 
-Als het aanwijshandje is aangeklikt als optie om de nieuwe variabele te definiëren, verschijnt er boven de data een veld met links daarvan de variabelen, boven wiskundige symbolen, en rechts een aantal functies. Hier kan handmatig de functie worden geselecteerd die de nieuwe variabele definieert. Om standaardscores te berekenen sleep je de variabele die je wilt omzetten in het lege veld. Klik dan bovenaan op $-$ en selecteer daarna uit het rechter menu de functie `mean(y)`. Sleep de om te zette variabele naar "values" en klik daarna bovenaan op $\div$. Selecteer in het rechter menu de functie `$\sigma$y` voor op de puntjes onder het breukteken (als noemer) en sleep ook hier de om te zette variabele naar "values". Voor de variabele 'Lezen' zou dit er zo uitzien: \[\frac{(Lezen-mean(Lezen)}{\sigma Lezen}\]
+Als het aanwijshandje is aangeklikt als optie om de nieuwe variabele te definiëren, verschijnt er boven de data een veld met links daarvan de variabelen, boven wiskundige symbolen, en rechts een aantal functies. Hier kan handmatig de functie worden geselecteerd die de nieuwe variabele definieert. Om standaardscores te berekenen sleep je de variabele die je wilt omzetten in het lege veld. Klik dan bovenaan op $-$ en selecteer daarna uit het rechter menu de functie `mean(y)`. Sleep de om te zette variabele naar "values" en klik daarna bovenaan op $\div$. Selecteer in het rechter menu de functie <span style="background-color:#f7f7f7"> $\sigma$y </span> voor op de puntjes onder het breukteken (als noemer) en sleep ook hier de om te zette variabele naar "values". Voor de variabele 'Lezen' zou dit er zo uitzien in het werkblad: \[\frac{(Lezen-mean(Lezen)}{\sigma Lezen}\]
```

Comment:
Is er een reden voor deze 
<span style="background-color:#f7f7f7">
html tag? Zo niet dan liever weglaten om er later geen last van te krijgen.